docs/business.yaml

```diff
@@ -287,6 +376,74 @@ paths:
                       courtOrder:
                         fileNumber: '12345'
                         effectOfOrder: planOfArrangement
+              Change-of-address-request:
```

Comment:
with the updated naming convention, the example name is all lower case, please refer to the examples in Teams group chat. e.g. `agm-location-change-request`